view_controllers/OBARegionListViewController.m

```diff
@@ -343,7 +343,13 @@ - (void)showLocationServicesAlert {
         UIAlertView *view = [[UIAlertView alloc] init];
         view.title = NSLocalizedString(@"Location Services Disabled", @"view.title");
         view.message = NSLocalizedString(@"Location Services are disabled for this app. Some location-aware functionality will be missing.", @"view.message");
-        [view addButtonWithTitle:NSLocalizedString(@"Dismiss", @"view addButtonWithTitle")];
+        view.delegate = self;
+        [view addButtonWithTitle:NSLocalizedString(@"Okay", @"Ok button")];
+
+        if (&UIApplicationOpenSettingsURLString != NULL) {
+            [view addButtonWithTitle:@"Location Settings"];
```

Comment:
`NSLocalizedString(@"Change Location Settings", @"Change location settings button for enabling location services.")`